packages/nimble/R/cppDefs_cppProject.R

```diff
@@ -268,17 +266,18 @@ cppProjectClass <- setRefClass('cppProjectClass',
                                        if(is.null(nimbleUserNamespace$sessionSpecificDll)) compileDynamicRegistrations(showCompilerOutput = showCompilerOutput)
 
                                        origSHLIBcmd <- SHLIBcmd
-                                       if(isTRUE(nimbleOptions()$stopCompilationBeforeLinking)) {## used only for testing, when we want to go quickly and skip linking and bail out
+                                       if(isTRUE(nimbleOptions('stopCompilationBeforeLinking'))) {## used only for testing, when we want to go quickly and skip linking and bail out
```

Comment:
Is there an intended difference between `getNimbleOption('myOption')` and `nimbleOptions('myOption')`? They are both used in this file.